src/Pagination/Footer/Footer.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import Buttons from './Buttons/Buttons'
 import Numbers from './Buttons/Numbers'
 
-const Footer = ({ paginationButtons, ...props }) => {
+const Footer = ({ paginationButtons, isSingleField, ...props }) => {
   const { type } = paginationButtons
 
-  if (type === 0)
-    return <Buttons paginationButtons={paginationButtons} {...props} />
+  if (type === 0 || isSingleField)
```

Comment:
Why we need this condition here ?

Reply:
If using single field Add-ons, builder show button with text.